core-bundle/src/Doctrine/Backup/Config/CreateConfig.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Doctrine\Backup\Config;
+
+class CreateConfig extends AbstractConfig
```

Comment:
```suggestion
class BackupConfig extends AbstractConfig
```
?

Reply:
Everything is Backup config but this one is the create config. So that's correct.